rabbitmq/ci/start-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source utils/bash/ci.sh
+
+NAME='dd-test-rabbitmq'
+
+stop-docker $NAME
```

Comment:
On Travis, yes that's safe to assume. On a local machine it's not so clear. I put this there because I was testing the containers and forgot to spin them down a number of times, so this would get rid of them rather than erroring out.